validphys2/src/validphys/tests/test_pythonmakereplica.py

```diff
@@ -38,14 +39,24 @@ def test_commondata_unchanged(data_config, dataset_inputs, use_cuts):
     config = dict(data_config)
     config["dataset_inputs"] = dataset_inputs
     config["use_cuts"] = use_cuts
+    config["replica_mcseed"] = SEED
+    config["use_t0_sampling"] = False
+    config["separate_multiplicative"]= True
+    config["output_path"] = None
+    config["theory_covmat_flag"] = False
+    config["use_user_uncertainties"] = None 
+    config["use_scalevar_uncertainties"] = None
+    config["use_thcovmat_in_sampling"]=None
+    config["dataset_t0_predictions"] = None
```

Comment:
Same as before. I don't like the fact that you need all these `None` and `False`. I need to check in the vp part but I think the problem is that you should not arrive to the functions in which you check that it is `False` or `None`.